types/dispatchr/addons/createStore.d.ts

```diff
@@ -0,0 +1,13 @@
+import { StoreClass, Store } from '../index';
+
+interface StoreOptions {
+    initialize?(): void;
+    storeName: string;
+    handlers: { [event: string]: string };
+    [prop: string]: any;
+}
+
+type CreateStore = <This extends StoreOptions>(options: This & ThisType<This & Store>) => StoreClass;
+
+declare const _: CreateStore;
```

Comment:
`declare function createStore <This extends StoreOptions>(...) => ...;`  
`export = createStore`